indico/modules/rb_new/client/js/common/rooms/RoomEditModal.jsx

```diff
@@ -0,0 +1,659 @@
+        type: 'header',
+        label: Translate.string('Information')
+    }, {
+        type: 'formgroup',
+        key: 'information',
+        content: [{
+            type: 'input',
+            name: 'capacity',
+            label: Translate.string('Capacity(seats)'),
+            inputType: 'number',
+            required: true
+        }, {
+            type: 'input',
+            name: 'division',
+            label: Translate.string('Division'),
+            required: false
+        }]
+    }, {
+        type: 'textarea',
+        name: 'comments',
+        label: Translate.string('Comments'),
+        required: false
+    }, {
+        type: 'header',
+        label: Translate.string('Daily availability'),
+    }, {
+        type: 'bookableHours',
+    }], [{
+        type: 'header',
+        label: Translate.string('Location')
+    }, {
+        type: 'input',
+        name: 'verboseName',
+        label: Translate.string('Name'),
+        required: false
+    }, {
+        type: 'input',
+        name: 'site',
+        label: Translate.string('Site'),
+        required: false
+    }, {
+        type: 'formgroup',
+        key: 'details',
+        content: [{
+            type: 'input',
+            name: 'building',
+            label: Translate.string('Building'),
+            required: true
+        }, {
+            type: 'input',
+            name: 'floor',
+            label: Translate.string('Floor'),
+            required: true
+        }, {
+            type: 'input',
+            name: 'number',
+            label: Translate.string('Number'),
+            required: true
+        }]
+    }, {
+        type: 'formgroup',
+        key: 'coordinates',
+        content: [{
+            type: 'input',
+            name: 'longitude',
+            label: Translate.string('longitude'),
+            inputType: 'number',
+            required: false
+        }, {
+            type: 'input',
+            name: 'latitude',
+            label: Translate.string('latitude'),
+            inputType: 'number',
+            required: false
+        }]
+    }, {
+        type: 'input',
+        name: 'surfaceArea',
+        label: Translate.string('Surface Area (m2)'),
+        inputType: 'number',
+        required: false
+    }, {
+        type: 'input',
+        name: 'maxAdvanceDays',
+        label: Translate.string('Maximum advance time for bookings (days)'),
+        inputType: 'number',
+        required: false
+    }, {
+        type: 'header',
+        label: 'Options',
+    }, {
+        type: 'checkbox',
+        name: 'isActive',
+        label: Translate.string('Active')
+    }, {
+        type: 'checkbox',
+        name: 'isReservable',
+        label: Translate.string('Bookable')
+    }, {
+        type: 'checkbox',
+        name: 'isAutoConfirm',
+        label: Translate.string('Confirmation')
+    }, {
+        type: 'checkbox',
+        name: 'notificationsEnabled',
+        label: Translate.string('Reminders Enabled')
```

Comment:
Perhaps it's a good idea that when this is unchecked, the booking reminder fields (`notificationBeforeDays`, etc.) look disabled.
This way you can just keep "Reminders" as label, as the user would have immediate feedback of what the field means.

Reply:
Nice idea. I think it makes sense to be implemented.